auth.py

```diff
@@ -0,0 +1,66 @@
+import logging
+
+from flask import (Blueprint, current_app, flash, redirect, render_template,
+                   request, url_for)
+from flask_login import login_user, login_required, logout_user
+from werkzeug.security import check_password_hash, generate_password_hash
+
+from db_operations import create_user, get_user
+from forms import LoginForm, SignupForm
+from users import User
+
+auth = Blueprint('auth', __name__)
+
+@auth.route('/login', methods=['GET', 'POST'])
+def login():
+    log = current_app.logger
```

Comment:
We might want to check first if the user is already logged in, and if so maybe just redirect them to the main page
We can do this by adding an if block in the beginning of the function
`if current_user.is_authenticated:`
will need to import current_user from flask_login first

Reply:
Ahh this makes sense! Updated!